packages/effects-runtime/readme.md

```diff
@@ -9,45 +9,40 @@ Performing, handling and recalling are handled by generators under the hood. The
 Consider the following variables
 
 ```javascript
-const getIntegerHandler = 'getInteger';
+const getIntegerHandler = "getInteger";
 
 const GetIntegerEffect = () => ({
-  type: getIntegerHandler
+  type: getIntegerHandler,
 });
 ```
 
 And compare the following code
 
 ```javascript
 const main = async () => {
-    'use effects';
-    try{
-        return perform GetIntegerEffect();
-
-        return integer
-    }handle getIntegerHandler with (e){
-        recall 5;
-    }
-}
+  "use effects";
+  try{
+    return perform GetIntegerEffect();
+  } handle getIntegerHandler with (e) {
+    recall 5;
+  }
+};
```

Comment:
this block wouldn't autoformat. you can tell because it's slightly inconsistent. not sure why prettier (or the effects plugin) didn't like it. for another time

Reply:
Hmmm